packages/codec/lib/abi-data/allocate/index.ts

```diff
@@ -1014,9 +1014,7 @@ function constructorOutputAllocation(
   if (contractKind === "library") {
     //note: I am relying on this being present!
     //(also this part is a bit HACKy)
-    const pushAddressInstruction = (0x60 + Evm.Utils.ADDRESS_SIZE - 1).toString(
-      16
-    ); //"73"
+    const pushAddressInstruction = (0x5f + Evm.Utils.ADDRESS_SIZE).toString(16); //"73"
```

Comment:
Note, this calculation is used in three places across two modules, adding error-prone maintenance complexity.

Reply:
There's not really any maintenance complexity here?  Like there's nothing here that needs to be updated.  I just changed the base of the calculation from `PUSH1` to `PUSH0` now that the latter exists... what would change it further in the future?

Note of course I could have just used a literal `"73"` but I figured that doing the calculation made the intent clearer.

Reply:
I meant that if a new opcode is updated to be base, future maintainers would have three places to update.